metal/cloud.go

```diff
@@ -4,11 +4,10 @@ import (
 	"fmt"
 	"io"
 
-	"github.com/packethost/packngo"
-
+	metal "github.com/equinix/equinix-sdk-go/services/metalv1"
 	cloudprovider "k8s.io/cloud-provider"
 	"k8s.io/component-base/version"
-	"k8s.io/klog/v2"
```

Comment:
This probably shouldn't change, but I don't know if there was an error or something that forced this change?

Reply:
It got changed via go mod tidy I guess. Reverted